results/cmd/api/main.go

```diff
@@ -35,11 +37,21 @@ func main() {
 	// Connect to sqlite DB.
 	db, err := sql.Open("sqlite3", "./results.db")
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalf("failed to open the results.db: %v", err)
 	}
 	defer db.Close()
 	srv := &server{db: db}
+	database := srv.db
 
+	// to-do: add more fields like name, namespace
+	statement, err := database.Prepare("CREATE TABLE IF NOT EXISTS taskrun (logid INTEGER PRIMARY KEY, taskrunlog BLOB)")
```

Comment:
I'm okay if you want to keep things this way for this PR, but we should look into separating the DB creation from the binary at some point in the near future.

Reply:
done